### PR TITLE
add news of Zedong's arrival and MindtPy abstract

### PR DESCRIPTION
**Pull request by ZedongPeng**

### CODE REVIEW

File: _posts/2023-08-01-Zedong.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: Welcome Dr. Peng
+description: 'The first postdoc in our group'
+image: assets/images/posts/Zedong_Albert.jpeg
+---
+
+Zedong Peng obtained his Ph.D. degree from Zhejiang University in 2021, co-advised by Gang Rong and Hongye Su. During his Ph.D, he visited the Grossmann Research Group at Carnegie Mellon University in 2018. In addition, Zedong is also the main developer of [MindtPy](https://pyomo.readthedocs.io/en/stable/contributed_packages/mindtpy.html). We are delighted to have Zedong join our group.
+
```
Comment (bernalde) [inline _posts/2023-08-01-Zedong.md:9]: This is a great description. Include as much more detail as you want (you are interested in pursuing a job in academia in China, you'll be working on MINLP algorithms and software)

File: _posts/2023-07-01-MindtPy.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: AIChE 2023 and INFORMS 2023
+description: 'The MindtPy abstract accecpted'
```
[the remark this answers is not in the document]
Comment (ZedongPeng) [reply _posts/2023-07-01-MindtPy.md:4]: Good suggestion. Thanks.

File: _posts/2023-03-08-APS.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 layout: post
 title: APS 2023 March Meeting
 description: 'My most recent conference attendance'
```
[the remark this answers is not in the document]
Comment (ZedongPeng) [reply _posts/2023-03-08-APS.md:4]: Should we change it to only "conference attendance"?

[the remark this answers is not in the document]
Comment (bernalde) [reply _posts/2023-03-08-APS.md:4]: Yes

File: _posts/2023-07-01-MindtPy.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: AIChE 2023 and INFORMS 2023
+description: 'The MindtPy abstract accepted'
+image: assets/images/posts/INFORMS2023.png
+---
+
+The abstract of MindtPy has been accepted to both [2023 AIChE Annual Meeting](https://www.aiche.org/conferences/aiche-annual-meeting/2023?gclid=Cj0KCQjwrMKmBhCJARIsAHuEAPR5h2LVn51eow9v9Vkcsy4C0goZVonySZxvBWmeif5ffqKY29GgfOYaAnNmEALw_wcB) and [2023 INFORMS Annual Meeting](https://meetings.informs.org/wordpress/phoenix2023/).
```
Comment (bernalde) [inline _posts/2023-07-01-MindtPy.md:8]: Link MindtPy to the documentation of Pyomo

File: _posts/2023-08-01-Zedong.md (hunk 1 above)
Comment (bernalde) [reply _posts/2023-08-01-Zedong.md:9]: Link your name to the page where your profile is

**Review by bernalde — CHANGES_REQUESTED** [review]

**Review by bernalde — APPROVED** [review]

**Pull request merged**